site/en/docs/react_gemini_prompting.ipynb

```diff
@@ -0,0 +1,973 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #34.        """helper function for responses
</pre><p> </p><p>Keep the style of docstrings uniform throughout the file. The style guide we follow requires a single (complete) sentence in the first line and paragraphs thereafter, so for this string it would be <code>"""Helper function for responses."""</code></p><p> </p><p>Following the style guide prescriptively isn't super important but being consistent within the file is. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/google/generative-ai-docs/pull/255/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='31'/>